brands/amenity/cinema.json

```
@@ -256,6 +256,15 @@
       "name": "Cinéville"
     }
   },
+  "amenity/cinema|Megarama": {
+    "locationSet": {"include": ["fr"]},
```

From http://megarama.fr/ it looks like we can add Spain and Morocco here:
```suggestion
    "locationSet": {"include": ["es", "fr", "ma"]},
```